shotover-proxy/src/transforms/chain.rs

```diff
@@ -115,43 +115,48 @@ impl TransformChain {
         // Even though we don't keep the join handle, this thread will wrap up once all corresponding senders have been dropped.
 
         let mut chain = self.clone();
-        let _jh = tokio::spawn(async move {
-            while let Some(ChannelMessage {
-                return_chan,
-                messages,
-            }) = rx.recv().await
-            {
-                #[cfg(test)]
+        let _jh = tokio::spawn(
```

Comment:
Very little was actually change here, the only thing I did here was call .in_current_span() but it made rust fmt freak out.